src/main/java/org/apache/paimon/trino/TrinoTableOptions.java

```diff
@@ -0,0 +1,460 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.paimon.trino;
+
+import io.trino.spi.session.PropertyMetadata;
+import io.trino.spi.type.ArrayType;
+import org.apache.flink.shaded.guava30.com.google.common.collect.ImmutableList;
+import org.apache.paimon.CoreOptions;
+import org.apache.paimon.WriteMode;
+
+import java.util.List;
+import java.util.Map;
+
+import static io.trino.spi.session.PropertyMetadata.booleanProperty;
+import static io.trino.spi.session.PropertyMetadata.doubleProperty;
+import static io.trino.spi.session.PropertyMetadata.enumProperty;
+import static io.trino.spi.session.PropertyMetadata.integerProperty;
+import static io.trino.spi.session.PropertyMetadata.longProperty;
+import static io.trino.spi.session.PropertyMetadata.stringProperty;
+import static io.trino.spi.type.VarcharType.VARCHAR;
+
+/** Trino table options */
+public class TrinoTableOptions {
+
+    public static final String FILE_FORMAT_PROPERTY = "file_format";
```

Comment:
Can we have a way just extract options from `CoreOptions`?
Just like `ConfigOptionsDocGenerator.extractConfigOptions`.

Reply:
ok, let me think about it